docs/entities-intro.rst

```diff
@@ -172,8 +190,10 @@ In general, workflows that are focused or short-lived can very easily be represe
 
 That said, in many contexts, workflow needs are so specific that a flexible platform like ODK offers great benefits. Once you have defined your workflow in ODK, the forms you have built can become the standard, specialized way to support others in your domain.
 
-Mechanics
-=========
+.. _entities-intro-get-started:
+
+Get started
```

Comment:
I like this title, but people are going to jump straight here. The "How do I use forms to create or update Entities?" might be a better first question then.